<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addservicedialog</class>
 <widget class="QDialog" name="addservicedialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>729</width>
    <height>709</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Service Request Page</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/img/img/icon2.png</normaloff>:/img/img/icon2.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">/*background-image: url(:/img/img/3.jpg);</string>
  </property>
  <widget class="QComboBox" name="servicecomboBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>460</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift</family>
     <pointsize>11</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid black;
</string>
   </property>
   <item>
    <property name="text">
     <string>Oil Change</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Brake Repair</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Battery Replacement</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tire Rotation</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Engine Diagnostics</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>AC Repair</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Wheel Alignment</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Full Inspection</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Car Wash</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Other</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>510</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift</family>
     <pointsize>11</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Service Date :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift</family>
     <pointsize>15</pointsize>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
color: rgb(0, 25, 255);
/*border: 4px solid black;
</string>
   </property>
   <property name="text">
    <string>Vehicle Info :-</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift</family>
     <pointsize>15</pointsize>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
color: rgb(0, 25, 255);
/*border: 4px solid black;
</string>
   </property>
   <property name="text">
    <string>Service Details:-</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="phonelineEdit">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>110</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift</family>
     <pointsize>11</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid black;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift</family>
     <pointsize>11</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Name   :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift</family>
     <pointsize>15</pointsize>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
color: rgb(0, 25, 255);
/*border: 4px solid black;</string>
   </property>
   <property name="text">
    <string>Customer Info :-</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="brandlineEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>280</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift</family>
     <pointsize>11</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid black;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>460</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift</family>
     <pointsize>11</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Service Type :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>460</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift</family>
     <pointsize>11</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Status :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="namelineEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>110</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift</family>
     <pointsize>11</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid black;
</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>280</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift</family>
     <pointsize>11</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string> Model  :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="addresslineEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>160</y>
     <width>601</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift</family>
     <pointsize>11</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift</family>
     <pointsize>11</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Address   :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>110</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift</family>
     <pointsize>11</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Phone No. :</string>
   </property>
  </widget>
  <widget class="QComboBox" name="statuscomboBox">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>460</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift</family>
     <pointsize>11</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid black;
</string>
   </property>
   <item>
    <property name="text">
     <string>New</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Pending</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Completed</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift</family>
     <pointsize>11</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Vehicle No.</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="vehiclenolineEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>330</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift</family>
     <pointsize>11</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid black;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="modellineEdit">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>280</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift</family>
     <pointsize>11</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid black;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift</family>
     <pointsize>11</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Brand :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnsave">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>590</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: rgb(57, 17, 255);
    color: white;                  /* White text */
    padding: 8px 16px;             /* Inner spacing */
    font-weight: bold;
border-radius: 20px ;
border:3px solid black;

 }
QPushButton:hover {
         /* Darker blue on hover */
	background-color: rgb(8, 160, 255);
}

QPushButton:pressed {
    background-color: #003d80;     /* Even darker on press */
}</string>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="servicedate">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>510</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift</family>
     <pointsize>11</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QCalenderWidget{
border:1px solid black;
}
QCalendarWidget QToolButton {
        color: black;              /* Month/Year text color */
        font-size: 16px;
        icon-size: 24px, 24px;
      
    }
    QCalendarWidget QMenu {
        background-color: white;
        color: black;
    }</string>
   </property>
   <property name="displayFormat">
    <string>dd/MM/yyyy</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
   <property name="date">
    <date>
     <year>2025</year>
     <month>6</month>
     <day>14</day>
    </date>
   </property>
  </widget>
  <widget class="QPushButton" name="btncancel">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>590</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: rgb(57, 17, 255);
    color: white;                  /* White text */
    padding: 8px 16px;             /* Inner spacing */
    font-weight: bold;
border-radius: 20px ;
border:3px solid black;

 }
QPushButton:hover {
         /* Darker blue on hover */
	background-color: rgb(8, 160, 255);
}

QPushButton:pressed {
    background-color: #003d80;     /* Even darker on press */
}</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>811</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift</family>
     <pointsize>22</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 3px solid black;
/*
background-color: rgb(255, 6, 31);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>                     Fill the Service Entry Form</string>
   </property>
  </widget>
  <zorder>label_3</zorder>
  <zorder>label_5</zorder>
  <zorder>servicecomboBox</zorder>
  <zorder>label_12</zorder>
  <zorder>label_7</zorder>
  <zorder>label_10</zorder>
  <zorder>phonelineEdit</zorder>
  <zorder>label</zorder>
  <zorder>label_6</zorder>
  <zorder>brandlineEdit</zorder>
  <zorder>label_4</zorder>
  <zorder>label_11</zorder>
  <zorder>namelineEdit</zorder>
  <zorder>label_8</zorder>
  <zorder>addresslineEdit</zorder>
  <zorder>label_2</zorder>
  <zorder>statuscomboBox</zorder>
  <zorder>label_9</zorder>
  <zorder>vehiclenolineEdit</zorder>
  <zorder>modellineEdit</zorder>
  <zorder>btnsave</zorder>
  <zorder>servicedate</zorder>
  <zorder>btncancel</zorder>
  <zorder>label_13</zorder>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
